src/WebJobs.Extensions.DurableTask/OverridableStates.cs

```diff
@@ -21,4 +24,33 @@ public enum OverridableStates
         /// </summary>
         NonRunningStates,
     }
+
+    /// <summary>
+    /// Extension methods for <see cref="OverridableStates"/>.
+    /// </summary>
+#pragma warning disable SA1649 // File name should match first type name Justification: pairing extension methods with enum.
+    internal static class OverridableStatesExtensions
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        private static readonly OrchestrationStatus[] NonRunning = new OrchestrationStatus[]
+        {
+            OrchestrationStatus.Running,
+            OrchestrationStatus.ContinuedAsNew,
+            OrchestrationStatus.Pending,
+        };
```

Comment:
Does `Suspended` need to be in this list?

Reply:
Yeah, I think that is a good change. Might even be a bug fix for all other DF languages